doc/marian-integration.md

```diff
@@ -15,32 +27,47 @@ sudo apt-get install intel-mkl-64bit-2020.0-088
 ```
 On MacOS, apple accelerate framework will be used instead of MKL/OpenBLAS.
 
+
+### Building bergamot-translator
+
+Web Assembly (WASM) reduces building to only using a subset of functionalities
+of marian, the translation library powering bergamot-translator. When
+developing bergamot-translator it is important that the sources added be
+compatible with marian.  Therefore, it is required to set
+`-DUSE_WASM_COMPATIBLE_SOURCE=on`.
+
 ```
 $ git clone https://github.com/browsermt/bergamot-translator
 $ cd bergamot-translator
 $ mkdir build
 $ cd build
 $ cmake .. -DUSE_WASM_COMPATIBLE_SOURCE=off -DCMAKE_BUILD_TYPE=Release
-$ make -j
+$ make -j4
```

Comment:
This requires at least 16GB of RAM, or it would fail, beware.

Reply:
Set to 2